core/bin/migrations/20241204_create_nodes.rs

```diff
@@ -29,25 +29,25 @@ enum NodeType {
 lazy_static! {
     static ref DOC_MIME_TYPES_MATCH: Vec<(fn(&str) -> bool, &'static str)> = vec![
         // Confluence
-        ((|s: &str| s.starts_with("confluence-page-")), "application/vnd.dust.confluence-page"),
+        ((|s: &str| s.starts_with("confluence-page-")), "application/vnd.dust.confluence.page"),
         // Github
-        ((|s: &str| s.starts_with("github-issue-")), "application/vnd.dust.github-issue"),
-        ((|s: &str| s.starts_with("github-discussion-")), "application/vnd.dust.github-discussion"),
-        ((|s: &str| s.starts_with("github-code-")), "application/vnd.dust.github-code-file"),
+        ((|s: &str| s.starts_with("github-issue-")), "application/vnd.dust.github.issue"),
+        ((|s: &str| s.starts_with("github-discussion-")), "application/vnd.dust.github.discussion"),
+        ((|s: &str| s.starts_with("github-code-")), "application/vnd.dust.github.codefile"),
```

Comment:
code.file I would use

Reply:
as we will have code.folders soon ?

Reply:
For now, it was planned that all folders would be `application/vnd.dust.folder`